File: Program/C++ prog/date.cpp
#include <stdio.h>

class Date
{
    int day;
    int month;
    int year;

  public:
    void setDay(int a)
    {
        day = a;
    }
    void setMonth(int b)
    {
        month = b;
    }
    void setYear(int c)
    {
        year = c;
    }

    int getDay()
    {
        return this->day;
    }
    int getMonth()
    {
        return this->month;
    }
    int getYear()
    {
        return this->year;
    }
    void display()
    {
        printf("Day is:%d month Is:%d Year is:%d", this->day, this->month, this->year);
    }
};

int main()
{
    Date d1;
    d1.setDay(12);
    d1.setMonth(8);
    d1.setYear(1996);
    d1.display();
}

File: Program/Java Program/Assignment_1/Program_5_Date/src/Date.java
public class Date {
    int day;
    int month;
    int year;

    Date() {
        day = 12;
        month = 8;
        year = 1996;
    }

    public void setDay(int day) {
        if (day > 0 && day <= 31)
            this.day = day;
        else
            System.out.println("invalid");
    }

    public void setMonth(int month) {
        if (month > 0 && month <= 12)
            this.month = month;
        else
            System.out.println("invalid");
    }

    public void setYear(int year) {
        this.year = year;
    }

    public int getDay() {
        return day;
    }

    public int getMonth() {
        return month;
    }

    public int getYear() {
        return year;
    }

    void display() {
        System.out.println("Date is : "+getDay()+"/"+getMonth()+"/"+getYear());
    }

}
